kalite/testing/mixins/browser_mixins.py

```diff
@@ -1,6 +1,7 @@
 import json
 import time
 import re
+import logging
```

Comment:
Quickfixing stuff is the main reason... can you import django.conf here? It seems to be outside of an initialized test env...